src/pass/arg_binder.cc

```diff
@@ -128,7 +128,7 @@ void ArgBinder::BindBuffer(const Buffer& arg,
     CHECK(fuzzy_match) << "Argument " << arg_name << " size mismatch";
     size_t diff = value->shape.size() - arg->shape.size();
     for (size_t i = 0; i < diff; ++i) {
-      CHECK(is_one(value->shape[i]))
+      CHECK(is_one(Simplify(value->shape[i])))
```

Comment:
is this necessary? as most cases we eagerly simplify the constants, while calling simplify is certainly more powerful, it would be great if we can move such simplification to the place that produces this expression.

Reply:
So I did hit into this assert, thats why the change else I dont try to eagerly fix things which are not a problem. Do you want me to reproduce the error and paste here?

Reply:
OK, we can do that as a followup PR if you can dig a bit further.